File: beginners/easy/008.find-large-number/206.saadman-galib.findLargeNumber.go
package main
import "fmt"


func main(){
	var a, b float64
	fmt.Scanln(&a)
	fmt.Scanln(&b)

	if a > b {
		fmt.Println(a, "is the largest")
	} else if b > a {
		fmt.Println(b, "is the largest")
	} else{
		fmt.Println("Both are equal")
	}
}

File: beginners/easy/008.find-large-number/206.saadman-galib.findLargeNumber.cpp
#include<iostream>
#include<iomanip>
using namespace std;

int main(){
    int a, b;
    cin >> a >> b;
    if (a > b){
        cout << fixed << setprecision(2) << a << " is the largest" << endl;
    }
    else if (b > a){
        cout << fixed << setprecision(2) << b << " is the largest" << endl;
    }
    else {
        cout << "Both are equal" << endl;
    }
    return 0;
}